dev-lang/go/go-1.14.12-r1.ebuild

```diff
@@ -30,12 +30,13 @@ DESCRIPTION="A concurrent garbage collected and typesafe programming language"
 HOMEPAGE="https://golang.org"
 
 LICENSE="BSD"
-SLOT="0/${PV}"
+SLOT="$(ver_cut 1-2)/${PV}"
 
 BDEPEND="|| (
 		dev-lang/go
 		dev-lang/go-bootstrap )"
-RDEPEND="!<dev-go/go-tools-0_pre20150902"
+RDEPEND="!<dev-go/go-tools-0_pre20150902
+	!dev-lang/go:1.15"
```

Comment:
You would also need to block slot 0 to force portage to unmerge currently installed versions.